src/OpenTelemetry.Api/Trace/Status.cs

```diff
@@ -158,7 +158,7 @@ internal Status(CanonicalCode canonicalCode, string description = null)
         /// <summary>
         /// Gets the canonical code from this status.
         /// </summary>
-        public CanonicalCode CanonicalCode { get; }
+        public StatusCanonicalCode StatusCanonicalCode { get; }
```

Comment:
I understand why rename the enum. Should we keep the property to be just CanonicalCode? It is already a part of a Status class.

Reply:
Same python file you are referring: https://github.com/open-telemetry/opentelemetry-python/blob/abdfa12dfd76613bef5cc5e9266439e983074bbc/opentelemetry-api/src/opentelemetry/trace/status.py#L172-L175

Reply:
Agree.